src/migration/1714633817175-SquadTotalPostsTrigger.ts

```diff
@@ -0,0 +1,65 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SquadTotalPostsTrigger1714633817175 implements MigrationInterface {
+  name = 'SquadTotalPostsTrigger1714633817175';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    queryRunner.query(`
+        CREATE OR REPLACE FUNCTION increment_squad_posts_count()
+          RETURNS TRIGGER
+          LANGUAGE PLPGSQL
+          AS
+        $$
+        BEGIN
+          UPDATE  source
+          SET     flags = jsonb_set(flags, '{totalPosts}', to_jsonb(COALESCE(CAST(flags->>'totalPosts' AS INTEGER) , 0) + 1))
+          WHERE   id = NEW."sourceId"
+          AND     type = 'squad';
```

Comment:
We decided on this just for performance? (Just meant maybe not a bad thing to have for all sources, but also not needed now)

Reply:
Yeah, we can probably do that. That would just mean we will recalculate the stats for all sources on launch to have its merits.

Reply:
Does RFC say anything about this? If not let's keep it as is and we can see next week when Drgaos is back where do we want to go. I don't have strong opinions if we want to have it on all sources. I would expect this kind of aggregations could be done by clickhouse in the future so that is the reason against calculating now for every source.

Reply:
Yeah would indeed roll out like this for now.

Reply:
Awesome. Thank you both.